### PR TITLE
Enable exposed name

### PR DESCRIPTION
**Pull request by mAAdhaTTah**
Supersedes #11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,17 @@ The first row is the entry file. This file is the file you need to add to your
 page with a `<script src="entry.js"></script>`. This also includes necessary
 boilerplate to load the other files.
 
+### Use exposed name
+
+The module could be defined in format `file_path:expose_name` and be loaded by `loadjs(['expose_name'])`.
```
Comment (arian) [inline README.md]: Should this `file_path:expose_name` be updated?

Comment (mAAdhaTTah) [inline README.md]: Yes, was just updating as you commented :).

**Comment by arian** [comment]
:+1:

File: index.js [hunk 2]
```diff
@@ -30,8 +30,15 @@ function partitionBundle(b, opts) {
   // require the modules from the map
   forOwn(opts.map, function(modules, file) {
     modules.forEach(function(mod, i) {
-      var id = bresolve.sync(mod, rOpts);
-      shortIDLabels[id] = mod;
+      if (typeof mod === 'string') {
+        mod = {
+          require: mod,
+          expose: mod
+        }
```
Comment (arian) [inline index.js:37]: Can you add a semicolon here?

See https://travis-ci.org/arian/partition-bundle/jobs/103460093#L326

**Comment by arian** [comment]
Or never mind...

**Comment by mAAdhaTTah** [comment]
You merged too fast! :)

Thanks for this. Would you mind tagging a new version to npm?

**Comment by arian** [comment]
Just published 2.3.0

**Comment by mAAdhaTTah** [comment]
:+1: thank you!

**Pull request merged**